### PR TITLE
KER-376 Sentry fixes

### PR DESCRIPTION
**Pull request by mikkojamG**
# KER-376 Sentry fixes
- Replace raven-js with @sentry/react
- Replace redux-raven-middleware with @sentry/react integration
- Ignore ResizeObserver loop errors
- fixing hearingEditor actions error catches
- cookieUtils getCookie check that cookie is an array

Azure changes:
https://dev.azure.com/City-of-Helsinki/kerrokantasi/_git/kerrokantasi-pipelines/pullrequest/9023

### CODE REVIEW

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1043-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://kerrokantasi-pr1043-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1043-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://kerrokantasi-pr1043-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1043-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://kerrokantasi-pr1043-ui.dev.hel.ninja :laughing::tada::tada::tada:

File: src/index.js [hunk 1]
```diff
@@ -1,23 +1,30 @@
...
-  try {
-    if (config.uiConfig && config.uiConfig.sentryDns) Raven.config(config.uiConfig.sentryDns).install();
-  } catch (err) {
-    // eslint-disable-next-line no-console
-    console.error(err);
-  }
+commonInit(() => {
+  if (config.uiConfig && config.uiConfig.sentryDns && config.uiConfig.sentryEnvironment) {
+    Sentry.init({
+      dsn: config.uiConfig.sentryDns,
```
Comment (arpargo) [inline src/index.js:17]: Typo?

**Review by arpargo — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (mikkojamG) [reply src/index.js]: Fixed'dd.

**Review by arpargo — APPROVED** [review]

**Comment by terovirtanen** [comment]
**KERROKANTASI-UI branch is deployed to platta: https://kerrokantasi-pr1043-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://kerrokantasi-pr1043-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Pull request merged**